mkinitcpio

```diff
@@ -14,6 +14,8 @@ _f_functions=functions
 _f_config=mkinitcpio.conf
 _d_hooks="$PWD/hooks:/usr/lib/initcpio/hooks:/lib/initcpio/hooks"
 _d_install="$PWD/install:/usr/lib/initcpio/install:/lib/initcpio/install"
+_d_flag_hooks=""
+_d_flag_install=""
```

Comment:
No need to assign the empty string here, just `_d_flag_{install,hooks}=` is fine

Reply:
Fix'd